crates/holochain/tests/tests/validate.rs

```diff
@@ -0,0 +1,246 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+
+use assert2::{assert, let_assert};
+use holochain::conductor::api::error::ConductorApiError;
+use holochain::conductor::CellError;
+use holochain::core::workflow::WorkflowError;
+use holochain::prelude::*;
+use holochain::sweettest::*;
+use holochain_state::source_chain::SourceChainError;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn call_to_validate_passes() {
```

Comment:
Same question about whether `validate` is tested. This one I'm less concerned about having a unit test for because it's run so often in tests that check validation

Reply:
I can remove it. I mainly added it so I could check that validate was called how I expected before writing other tests and wondering why validate was never called :slightly_smiling_face:. It seemed harmless to leave in but it's not really a good test for true validation as it just always returns `Valid`.

Reply:
That's fair enough, I'm convinced about keeping the `init` one so let's keep this one too. I think maybe I've got a bit too used to really high value functional tests and I'm not appreciating small tests enough here :)

Reply:
It seems that `validate` is called automatically after calling `init`. Maybe there is a better test here to make the validation called by `init` pass and the `validate` called by a zome call fail.

Reply:
I find confusing that `validate` is run after `init` even though no entries are created during `init`.